.github/workflows/lima_upload_oidc.yml

```
@@ -1,12 +1,10 @@
 name: build and publish images for use with lima-vm
 
 on:
-  push:
-    branches:
-      - lima-upload  
   schedule:
     - cron: '0 11 * * 2'
-    
+  workflow_on_dispatch:
```

```suggestion
  workflow_dispatch:
```

are you testing how well I review?

for reference: https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#workflow_dispatch